README.md

```diff
@@ -307,6 +307,7 @@ Right now, Lyra supports 24 languages and stemmers out of the box:
 - Swedish
 - Turkish
 - Ukrainian
+- Slovenian
```

Comment:
Can you also update the `Right now, Lyra supports 24 languages and stemmers out of the box:` text to `25 languages`?

Reply:
Yes, I will update it shortly.